Add notes about deprecation to lib-content #443 (#444)
enonic/doc-xp: docs/auditLogs.adoc

* `system.content.archive` - a content has been archived
* `system.content.restore` - a content has been restored
* `system.content.rename` - a content has been renamed

> * `system.content.setActiveContentVersion` - an activeContentVersion has been set for the content. (Deprecated from image:xp-7120.svg[XP 7.12.0,opts=inline])

Reviewer: ```suggestion
* `system.content.setActiveContentVersion` - an activeContentVersion has been set for the content. (Not in use from image:xp-7120.svg[XP 7.12.0,opts=inline])
```

> * `system.content.setActiveContentVersion` - an activeContentVersion has been set for the content. (No longer supported image:xp-7120.svg[XP 7.12.0,opts=inline])

Reviewer: ```suggestion
* `system.content.setActiveContentVersion` - an activeContentVersion has been set for the content. (Deprecated from image:xp-7120.svg[XP 7.12.0,opts=inline])
```
